build.gradle

```diff
@@ -189,29 +194,14 @@ if(skipPrebuildLibraries!="true"&&buildNativeProjects!="true"){
                     String p1=srcRel.substring(0,j);
                     String p2=srcRel.substring(j);
                     if(!p1.equals("android")&&!p2.startsWith("/native"))  srcRel=p1+"/native"+p2;
-                    //
-
                     File dest=new File(nativesPath+File.separator+srcRel);
-
-                    boolean include=false;
-                    if(!dest.exists()){
-                        include=true;
-                        println("Copy "+src+" "+dest);
-                    }else if(dest.lastModified()<src.lastModified()){
-                        include=true;
-                        println("Copy "+src+" "+dest+  ". Source is newer. src "+src.lastModified()+ " dest "+dest.lastModified());
```

Comment:
Are you certain, that the copy mode (especially REPLACE_EXISTING) honors this behavior, that means: Can I have a locally built version of the natives which WONT be overridden then with the files from the zip?